backfill.py

```diff
@@ -49,7 +65,7 @@ def parse_args():
     return parser.parse_args()
 
 
-def fetch_nightly(download_date):
+def fetch_nightly(download_date, architecture):
     download_date_string = datetime.strftime(download_date, DATETIME_FORMAT)
     nightly_url = NIGHTLY_BASE_URL.format(date=download_date_string)
     filename = "nightly_" + download_date_string.replace(".", "_") + ".apk"
```

Comment:
nitpick (non-blocking): We're not actually using the architecture argument right now at all.

suggestion: to make things simpler, should we remove it from the arg parser and its use in the code and re-add it in a subsequent PR? I notice a few other issues with it too (e.g. a typo and inconsistencies in the argparse options).